### ISSUE TITLE
integration with LSL ?  How to send data to LSL?

### ISSUE THREAD

**Issue by 99bits** [kylemath/EEGEdu#184]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by kylemath** [comment]
can you explain what you are trying to do?

**Comment by 99bits** [comment]
I am using an app that uses LSL and wondering how to send data from RxJS to the LSL app.

**Comment by kylemath** [comment]
what is the app running on?

**Comment by 99bits** [comment]
it's a unity3d app

**Comment by 99bits** [comment]
https://github.com/luisqtr/exciteometer

**Comment by 99bits** [comment]
The app uses LSL and uses LSL libraries so it doesn't work for web.  However if  RxJS stream is a valid "observable" LSL stream then it would be fantastic.

**Comment by kylemath** [comment]
im not sure, but have used this alot prior to moving to web: https://github.com/alexandrebarachant/muse-lsl

**Comment by 99bits** [comment]
Ah, that uses the python version of LSL.

**Comment by kylemath** [comment]
yes is there a reason you need to run the LSL EEG stream from the web?

**Comment by 99bits** [comment]
Yes, because the app is also a unity webgl app that runs in the browser.
